server/rbacpolicy/rbacpolicy.go

```diff
@@ -126,10 +129,19 @@ func (p *RBACPolicyEnforcer) EnforceClaims(claims jwt.Claims, rvals ...interface
 	}
 	// Finally check if any of the user's groups grant them permissions
 	groups := jwtutil.GetScopeValues(mapClaims, scopes)
-	for _, group := range groups {
-		vals := append([]interface{}{group}, rvals[1:]...)
-		if p.enf.EnforceRuntimePolicy(runtimePolicy, vals...) {
-			return true
+
+	// Get groups to reduce the amount to checking groups
+	groupingPolicies := enforcer.GetGroupingPolicy()
```

Comment:
Awesome. Glad you did not have to implement your own policy parsing to get the list of groups.